lib/linguist/generated.rb

```diff
@@ -325,29 +327,13 @@ def node_modules?
       !!name.match(/node_modules\//)
     end
 
-    # Internal: Is the blob part of the Go vendor/ tree,
-    # not meant for humans in pull requests.
-    #
-    # Returns true or false.
-    def go_vendor?
-      !!name.match(/vendor\/((?!-)[-0-9A-Za-z]+(?<!-)\.)+(com|edu|gov|in|me|net|org|fm|io)/)
-    end
-
     # Internal: Is the blob a generated npm shrinkwrap or package lock file?
     #
     # Returns true or false.
     def npm_shrinkwrap_or_package_lock?
       name.match(/npm-shrinkwrap\.json/) || name.match(/package-lock\.json/)
     end
 
-    # Internal: Is the blob part of Godeps/,
-    # which are not meant for humans in pull requests.
-    #
-    # Returns true or false.
-    def godeps?
-      !!name.match(/Godeps\//)
-    end
```

Comment:
'scuse my ignorance, but why is this being removed? Is it because all `Godeps` will _always_ match `Code generated .* DO NOT EDIT` implemented above?

Reply:
Because vendored files are not generated files. Vendoring is already covered by https://github.com/github/linguist/blob/master/lib/linguist/vendor.yml#L51